CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Added
 
+-  Added `Operator Version: X.Y.Z` information in the operator logs.([#1953](https://github.com/operator-framework/operator-sdk/pull/1953))
```

Comment:
Nit: there is an extraneous space at the beginning and a missing space after the sentence-ending period.
```suggestion
- Added `Operator Version: X.Y.Z` information in the operator logs. ([#1953](https://github.com/operator-framework/operator-sdk/pull/1953))
```

Reply:
Fixed, thanks.